### PR TITLE
Update DeployWithDockerOnLinuxMac.md

### PR DESCRIPTION
**Pull request by hlebkanonik**

### CODE REVIEW

File: docs/installation-steps/DeployWithDockerOnLinuxMac.md [hunk 1]
```diff
@@ -94,10 +57,48 @@ Useful commands:
...
 * Default User: `default\1q2w3e`
 * Administrator: `superadmin\erebus`
 
 > ⚠️ Please change the admin password for better security
+
+## Optional Customisation
+
+1. Expose Docker Volumes to the file system
+
+>    * OPTIONAL: Set {vm.max_map_count} kernel setting before ReportPortal deploying with the following [Commands](https://www.elastic.co/guide/en/elasticsearch/reference/7.10/docker.html#docker-cli-run-prod-mode)
```
Comment (AmsterGet) [inline docs/installation-steps/DeployWithDockerOnLinuxMac.md:72]: Please remove asterisk and empty spaces here.

File: docs/installation-steps/DeployWithDockerOnLinuxMac.md [hunk 2]
```diff
@@ -94,10 +57,48 @@ Useful commands:
...
+
+Give the right permissions to the ElasticSearch data folder using the following commands:
+
+```bash
+mkdir -p data/elasticsearch
+chmod 777 data/elasticsearch
+chgrp 1000 data/elasticsearch
+```
+
+> For more details about ElasticSearch visit ElasticSearch [guide]> (https://www.elastic.co/guide/en/elasticsearch/reference/7.10/docker.html#_notes_for_production_use_and_defaults)
```
Comment (AmsterGet) [inline docs/installation-steps/DeployWithDockerOnLinuxMac.md:82]:
```suggestion
> For more details about ElasticSearch visit ElasticSearch [guide](https://www.elastic.co/guide/en/elasticsearch/reference/7.10/docker.html#_notes_for_production_use_and_defaults)
```

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: docs/installation-steps/DeployWithDockerOnLinuxMac.md (hunk 1 above)
Comment (hlebkanonik) [reply docs/installation-steps/DeployWithDockerOnLinuxMac.md]: done

**Review by hlebkanonik — COMMENTED** [review]
done

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**